dbt/logger.py

```diff
@@ -44,6 +43,7 @@
 
 
 def make_log_dir_if_missing(log_dir):
+    import dbt.clients.system
     dbt.clients.system.make_directory(log_dir)
```

Comment:
the logger imports the system client, and the system client imports the logger. This breaks the circular import